basedtyping/__init__.py

```diff
@@ -37,10 +38,11 @@
 # TODO: `Final[Literal[False]]` but basedmypy will whinge on usages
 #  https://github.com/KotlinIsland/basedmypy/issues/782
 BASEDMYPY_TYPE_CHECKING: Final = False
-"""a special constant, is always `False`, but basedmypy will always assume it to be true
+BASEDPYRIGHT_TYPE_CHECKING: Final = False
+"""special constants, atr always `False`, but basedmypy will always assume it to be true
```

Comment:
basedpyright doesnt support this yet
```suggestion
BASEDPYRIGHT_TYPE_CHECKING: Final = False
"""special constant, is always `False`, but basedpyright will always assume it to be true.

note that this currently does not work, but a future release of basedpyright will support it. see https://github.com/DetachHead/basedpyright/issues/50
```

Reply:
dropped it